ci/jobs/rebase_analysis.yaml

```diff
@@ -0,0 +1,51 @@
+# This Jenkins job runs a script which analyzes the gap between upstream and downstream by querying
+# Redmine and Bugzilla. See https://github.com/pulp/pulp_packaging/ci/redmine_bugzilla.py
+
+- job-template:
```

Comment:
I copied this from the redmin.yaml which provides the redmine-bugzilla-automation job. I updated this new job and that one to be s/-template//.